### PR TITLE
Documentation for Drum Push

### PR DESCRIPTION
**Pull request by 4dahalibut**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
When I pushed out the code I didn't write docs. This also fixes some bugs.

### CODE REVIEW

**Comment by yakov-g** [comment]
What about drum's readme? The one in custom_model_runner folder

**Comment by 4dahalibut** [comment]
What is the difference between the two readmes.

**Comment by yakov-g** [comment]
Readme from custom_model_runner is shown in PyPi. It should have just a short description and, if needed, point to the main readme.

File: custom_model_runner/datarobot_drum/drum/drum.py [hunk 1]
```diff
@@ -554,11 +551,12 @@ def maybe_build_image(self, docker_image_or_directory):
         try:
             image, _ = client.images.build(path=docker_image_or_directory)
         except docker.errors.BuildError as e:
-            self.logger.error("Hey something went wrong with image build!")
             for line in e.build_log:
                 if "stream" in line:
                     self.logger.error(line["stream"].strip())
-            raise
+            raise DrumCommonException(
+                "There was an error while building your image. " "Check the above logs to debug. "
```
Comment (yakov-g) [inline custom_model_runner/datarobot_drum/drum/drum.py:558]: Too many quotes.
Fix if there will be more fixes.

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**2.19** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**